server/src/internalClusterTest/java/org/opensearch/discovery/StableClusterManagerDisruptionIT.java

```diff
@@ -161,38 +161,38 @@ private void testFollowerCheckerAfterMasterReelection(NetworkLinkDisruptionType
         internalCluster().startNodes(4, settings);
         ensureStableCluster(4);
 
-        logger.info("--> stopping current master");
+        logger.info("--> stopping current cluster-manager");
         internalCluster().stopCurrentMasterNode();
 
         ensureStableCluster(3);
 
-        final String master = internalCluster().getMasterName();
-        final List<String> nonMasters = Arrays.stream(internalCluster().getNodeNames())
-            .filter(n -> master.equals(n) == false)
+        final String clusterManager = internalCluster().getMasterName();
+        final List<String> nonClusterManagers = Arrays.stream(internalCluster().getNodeNames())
+            .filter(n -> clusterManager.equals(n) == false)
             .collect(Collectors.toList());
-        final String isolatedNode = randomFrom(nonMasters);
-        final String otherNode = nonMasters.get(nonMasters.get(0).equals(isolatedNode) ? 1 : 0);
+        final String isolatedNode = randomFrom(nonClusterManagers);
+        final String otherNode = nonClusterManagers.get(nonClusterManagers.get(0).equals(isolatedNode) ? 1 : 0);
 
         logger.info("--> isolating [{}]", isolatedNode);
 
         final NetworkDisruption networkDisruption = new NetworkDisruption(
-            new TwoPartitions(singleton(isolatedNode), Sets.newHashSet(master, otherNode)),
+            new TwoPartitions(singleton(isolatedNode), Sets.newHashSet(clusterManager, otherNode)),
             networkLinkDisruptionType
         );
         setDisruptionScheme(networkDisruption);
         networkDisruption.startDisrupting();
 
-        logger.info("--> waiting for master to remove it");
-        ensureStableCluster(2, master);
+        logger.info("--> waiting for cluster-manager to remove it");
+        ensureStableCluster(2, clusterManager);
         ensureNoMaster(isolatedNode);
 
         networkDisruption.stopDisrupting();
         ensureStableCluster(3);
     }
 
     /**
-     * Tests that emulates a frozen elected master node that unfreezes and pushes its cluster state to other nodes that already are
-     * following another elected master node. These nodes should reject this cluster state and prevent them from following the stale master.
+     * Tests that emulates a frozen elected cluster-manager node that unfreezes and pushes its cluster state to other nodes that already are
+     * following another elected cluster-manager node. These nodes should reject this cluster state and prevent them from following the stale cluster-manager.
      */
     public void testStaleMasterNotHijackingMajority() throws Exception {
```

Comment:
`testStaleMasterNotHijackingMajority` -> `testStaleClusterManagerNotHijackingMajority`?

Reply:
Changed the method name in the new commit https://github.com/opensearch-project/OpenSearch/pull/2521/commits/02af408bcf8bf72ebff9f3a01592cf3cf0e9f49f.